tests/workflow/interfaces/qnode/test_tensorflow_qnode.py

```diff
@@ -1054,6 +1054,7 @@ def test_gradient_expansion(self, dev, diff_method, grad_on_execution, device_vj
 
         class PhaseShift(qml.PhaseShift):
             grad_method = None
+            has_generator = False
```

Comment:
Should we expect any user code to break due to this enhancement? :sweat_smile:

Reply:
So this is a weird situation where there is a defined generator, but it is correct for the decomposition. So this would only break user's code if they had an incorrect generator.

Reply:
I was thinking that the upgrade is a breaking change in that a new gradient method becomes "activated" if a generator is available, which might lead to unforeseen changes in behaviour, for example if a user relied on no method being available :sweat_smile: 
I don't know, we could mention the change in the breaking changes section, hypothetically...
https://xkcd.com/1172/

Reply:
I think this change broke the test only because we were writing the test in a rather convoluted way. I think this is strictly adding beneficial behavior, even if things happen in a slightly different way.